web/client/plugins/mapEditor/DefaultConfiguration.js

```diff
@@ -0,0 +1,148 @@
+export default {
```

Comment:
could this be a json file ?

Reply:
> While testing it manually i found the following:
> 
> * [ ]  identify point is different from the clicked one
>   ![image](https://user-images.githubusercontent.com/11991428/69543844-bb473480-0f8e-11ea-98a5-a73c87c55cbb.png)

I'm not able to replicate, may you give more info?



> * [x]  configure search services should be hidden here, since it does nothing
>   ![image](https://user-images.githubusercontent.com/11991428/69543957-fc3f4900-0f8e-11ea-9d5e-acf269d18392.png)

Fixed z-index, It's an issue requirement.

> * [ ]  it seems that preview zoom level is a level different from the one in the editor or when applied

> * [x]  actually the maps created within the story are stored in the same list "current story" where also the maps coming from an external services are stored- @offtherailz was this intended (i think yes) ?
Yes it is

Reply:
Yes do you prefer ad json file?